source/reference/ulimit.txt

```diff
@@ -303,22 +351,23 @@ form:
 
 .. _proc-file-system:
 
-``/proc`` File System
+``proc`` File System
 ~~~~~~~~~~~~~~~~~~~~~
 
 .. note::
 
    This section applies only to Linux operating systems.
 
-The ``/proc`` file-system stores the per-process limits in the
-file system object located at ``/proc/<pid>/limits``, where ``<pid>``
-is the process's :term:`PID` or process identifier. You can use the
-following ``bash`` function to return the content of the ``limits``
-object for a process or processes with a given name:
+The per-process limits in effect for a particular process can be obtained
+via the ``proc`` file system by examining the file located at
```

Comment:
Singular "file" makes me think that I am checking one config file for all processes. I would recommend something like:

" ... can be obtained via the ``proc`` filesystem. Each running process stores its current ``ulimit`` values in a ``limits`` file located at ``/proc/<pid>/limits``, where ... "